src/components/Icon/assets/royaltyShares.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
remove sizes and fill to have flexible size

Reply:
thanks